### ISSUE TITLE
Channel Name & Content Title is null in content_summary_logs.csv

### ISSUE THREAD

**Issue by hani647** [learningequality/kolibri#9601]
## Observed behavior

Channel Name & Content title seems to be missing in kolibri v15.6 in content_summary_logs.csv report. For previous version
it seems to be there but not sure what happened with newer version?

## Errors and logs
<!--
Relevant logs from:
 * the command line
 * ~/.kolibri/logs/kolibri.txt
 * the browser console

Please wrap errors in triple backticks for clean formatting like this:

01:10 info: something happened
01:12 error: something bad happened
```
-->


## Expected behavior
<!--
Description of what behavior was expected but did not occur
-->
content_summary_logs.csv must have "Content title" so that admin can see the time spent by the user on any specific item.
…

## User-facing consequences
<!--
Implications and real-world consequences for learners, coaches, admins, and other users of the application
-->
learner

## Steps to reproduce
<!--
Precise steps that someone else can follow in order to see this behavior
-->
1. Generate "content_summary_logs.csv".
2. Check content_summary_logs.csv 
3. You will see Channel Name & Content Title is empty.

## Context
<!--
Tell us about your environment, including:
 * Kolibri version
 * Operating system
 * Browser
-->
Kolibri: v15.x+

**Comment by rtibbles** [comment]
Hi @hani647 - thanks for reporting this. We've had a similar report from a user on our community forums: https://community.learningequality.org/t/kolibri-session-log-files-empty-system-glitches/2513/8 but have so far been unable to replicate the error in our local testing.

I am wondering if there's anything specific to your context that we might be missing? What language are the Channel titles and Resource titles in? What script are they using? Is there anything else that might be relevant that we may be missing?

**Comment by rtibbles** [comment]
Also - do you happen to know what Python version is being used to run Kolibri?

**Comment by hani647** [comment]
Hey @rtibbles ,
Thanks for the prompt response, I'm using Python 2.7.18.

**Comment by hani647** [comment]
Content title & Channel name are in English. We're using below endpoint to generate content_summary_logs.csv on Kolibri server and that causing the issue that's mentioned in your forum issue too.

/api/tasks/tasks/startexportlogcsv/

**Comment by hani647** [comment]
@rtibbles its worth to mention here that same code works well with kolibri v13.x and that column empty issue only exist in v15.x.

**Comment by rtibbles** [comment]
Thanks for the detailed feedback here - we think we've isolated the issue to a regression that occurred in the 0.15.5 release. In an effort to generate CSV files that were not vulnerable to injection attacks when opened in Microsoft Excel, it seems that these columns ended up getting removed instead of just santized.

**Comment by rtibbles** [comment]
This has now been fixed in #9602 - we are planning to make our next patch release by the end of August which will include this fix. Thanks for reporting!
